django_cloud_deploy/crash_handling/__init__.py

```diff
@@ -24,17 +24,24 @@
 
 import jinja2
 
+import django_cloud_deploy
 from django_cloud_deploy.cli import io
 from django_cloud_deploy import __version__
 
 
+class UserError(Exception):
+    """Error caused by user's code."""
+
+
+_KNOWN_EXCEPTIONS = ('UserError',)
```

Comment:
```suggestion
# A list of exceptions that should be displayed to the user rather than opening
# a Github issue.
_DISPLAYABLE_EXCEPTIONS = [UserError]
```

Reply:
Done.